bindata/etcd/cluster-backup-pod.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: cluster-backup
+  namespace: openshift-etcd
+spec:
+  containers:
+  - name: cluster-backup
+    imagePullPolicy: IfNotPresent
+    terminationMessagePolicy: FallbackToLogsOnError
+    command:
+      - /bin/sh
+      - -c
+      - |
+        #!/bin/sh
+        /usr/local/bin/cluster-backup.sh /etc/kubernetes/static-pod-resources/cluster-backup
+
+    resources:
+      requests:
+        memory: 60Mi
+        cpu: 10m
+    securityContext:
+      privileged: true
+    volumeMounts:
+      - mountPath: /usr/local/bin
+        name: usr-local-bin
+      - mountPath: /etc/kubernetes/static-pod-resources
+        name: resource-dir
+      - mountPath: /etc/kubernetes/static-pod-certs
+        name: cert-dir
+      - mountPath: /etc/kubernetes/manifests
+        name: static-pod-dir
+  priorityClassName: system-node-critical
+  nodeSelector:
+    node-role.kubernetes.io/master: ""
+  restartPolicy: Never
+  hostNetwork: true
+  tolerations:
+  - operator: "Exists"
```

Comment:
I'm not all that familiar with tolerations, but from [the docs][1], I'd expect `key` and `effect` here too.  Is a bare `Exists` something that should be documented as a special case in [our conventions][2]?

[1]: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/
[2]: https://github.com/openshift/enhancements/blob/39336b3bacc86aa28c01cec5b2a1b9fcd7de81e1/CONVENTIONS.md#taints-and-tolerations

Reply:
to be honest it was cargo culted from etcd operand as I threw the pod spec together. I will as a follow-up clean up the spec and ensure tolerations etc are sane.